From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: SeekingBlues <seekingblues@gmail.com>
Date: Fri, 3 Jun 2022 16:05:10 -0500
Subject: [PATCH] Do not define ESTALE in config.h.in

In ccache, `config.h` is included by the `-include` compiler option,
which means that it is included before any libc headers. This doesn't
cause any problems on systems that redefine this macro when libc headers
are included, but on Serenity, it breaks our definition of the
`ErrnoCode` enum.
---
 cmake/config.h.in | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/cmake/config.h.in b/cmake/config.h.in
index f39a18fc1b42dd095336da037ad751b741a5aae8..f599e924af43e194a4b55b2505dc4438612a9347 100644
--- a/cmake/config.h.in
+++ b/cmake/config.h.in
@@ -205,10 +205,6 @@ typedef int pid_t;
 #  define O_TEXT 0
 #endif
 
-#ifndef ESTALE
-#  define ESTALE -1
-#endif
-
 #define SYSCONFDIR "@CONFIG_SYSCONFDIR_C_ESCAPED@"
 
 #cmakedefine INODE_CACHE_SUPPORTED
